packages/owl-bot/src/bin/commands/scan-configs.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import admin from 'firebase-admin';
+import {readFile} from 'fs';
+import {FirestoreConfigsStore} from '../../database';
+import {scanGithubForConfigs} from '../../handlers';
+import {
+  getAuthenticatedOctokit,
+  getGitHubShortLivedAccessToken,
+} from '../../core';
+import {promisify} from 'util';
+import yargs = require('yargs');
+
+const readFileAsync = promisify(readFile);
+
+interface Args {
+  'pem-path': string;
+  'app-id': number;
+  installation: number;
+  org: string;
+  project: string;
+}
+
+export const scanConfigs: yargs.CommandModule<{}, Args> = {
```

Comment:
Added this helper so that I could attempt running the end to end scan locally to help me debug.